ops/main.py

```diff
@@ -174,7 +174,10 @@ def main(charm_class):
         juju_event_name = '{}_action'.format(juju_event_name)
 
     model_backend = ops.model.ModelBackend()
-    setup_root_logging(model_backend)
+    debug = False
+    if os.getenv("JUJU_DEBUG"):
+        debug = True
+    setup_root_logging(model_backend, debug=debug)
```

Comment:
Instead of getting the value and evaluating None to False, what about just doing `debug = JUJU_DEBUG in os.environ`?

Reply:
I kind of wanted both 'None' and '' to be treated as False, which is why I went this route. (JUJU_DEBUG='' is also not indicating debug is set.)
If you feel strongly, I can happily do this differently.

Reply:
I went ahead and switched it for now.